apps/creds/migrations/0001_initial.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 2.2.24 on 2021-12-07 17:26
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.OAUTH2_PROVIDER_APPLICATION_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ProdCredentialingReqest',
+            fields=[
+                ('creds_request_id', models.UUIDField(default=uuid.uuid4, editable=False, help_text='A UUID assigned to a prod credentialing request', primary_key=True, serialize=False, unique=True, verbose_name='Prod Credentialing Request ID')),
+                ('date_created', models.DateTimeField(auto_now_add=True)),
+                ('date_fetched', models.DateTimeField(blank=True, null=True)),
```

Comment:
What is "date_fetched"? How does it differ from "last_visit"? Also it might be useful to have an updated_at field (fairly common): `updated_at = models.DateTimeField(auto_now=True)` not sure if it would replace one of those fields

Reply:
Looking through how it's used, I think `date_fetched`, can just become `updated_at`. It'll give us the thing in a more common nomenclature.

Reply:
the intended use of 'date_fetched' is to help enforce fetch (download) creds once policy (more secure by restrictive)

Reply:
Hmmm don't think that fully answers my question. I wanted to know the difference between `date_fetched` and `last_visit`. I feel like you can just create a field called `updated_at` and whenever the row is updated (which it will be for the `visits_count` every time). We are using the created at time to determine the 5-10 minute acceptable window. We are using the `visits_count` to only let them generate the credentials once. Not sure what else these two fields are doing. `updated_at` is good for audit purposes, so we can see the last time a change was made when investigating. I'm open to having them, I'm just not understanding the benefit to having them there.